com.nitrokey.nitrokey-app.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.nitrokey.nitrokey-app
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: nitrokey-app
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=host
+  - --device=all
+modules:
```

Comment:
You should add a global `cleanup` array to remove unneeded files, something like this is a good starting point:

```
cleanup:
  - "*.a"
  - /include
```

Reply:
done and added more